admin-cli/cmd/table_migrator.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package cmd
+
+import (
+	"github.com/apache/incubator-pegasus/admin-cli/executor/toolkits/tablemigrator"
+	"github.com/apache/incubator-pegasus/admin-cli/shell"
+	"github.com/desertbit/grumble"
+)
+
+func init() {
+	shell.AddCommand(&grumble.Command{
+		Name: "table-migrator",
+		Help: "migrate table from current cluster to another via table duplication and metaproxy",
```

Comment:
Is the migrate tool bind to metaproxy? AFAIK, metaproxy is used for unify table access, how can it effect the migrate tool?

Reply:
migrate table actually contain two step:
1. server side copy table from current to remote
2. client switch connection with current cluster to remote

if user access via metaproxy, server side can auto-control client's target cluster addr and client no need update config 
 and restart service.

But using metaproxy and auto-switch is optional, user can just complete `step 1` and manual-switch table addr.

The tool has little complex,  I will add `readme` to explain its usage